pkgs/development/python-modules/cupy/default.nix

```diff
@@ -41,16 +42,27 @@ let
 in
 buildPythonPackage rec {
   pname = "cupy";
-  version = "13.0.0";
+  version = "13.2.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-LwTnhX9pKnEzYNycOwZwmAarhAT8o5ta+XIcBKKXmq4=";
+  src = fetchFromGitHub {
+    owner = "cupy";
+    repo = "cupy";
+    rev = "v13.2.0";
+    hash = "sha256-vZAtpIZztmsYeJeuq7yl7kgZse2azrIM3efHDmUswJI=";
+    fetchSubmodules = true;
```

Comment:
Thanks. Can you clarify which submodules are being used?

Reply:
I think it's only the three in here: https://github.com/cupy/cupy/tree/main/third_party, these are required to build, and are vendored by the pypi release.